package.json

```diff
@@ -5,14 +5,20 @@
     "build": "turbo build",
     "dev": "turbo dev",
     "lint": "turbo lint",
-    "format": "turbo format"
+    "format": "turbo format",
+    "test": "vitest"
```

Comment:
The turborepo paradigm here is to set this to `turbo test`, update the `turbo.json` file with:

```
"test": {
  "dependsOn": ["^test"]
},
```

 and then add a "test": "vitest" to each workspace that needs to run tests.